en_US/data-integration/azure-blob-storage.md

```diff
@@ -0,0 +1,196 @@
+
+After events and message data are written to Storage Container, you can connect to Azure Blob Storage to read the data for flexible application development, such as:
+
+- Data archiving: Store device messages as objects in Azure Blob Storage for long-term preservation to meet compliance requirements or business needs.
+- Data analysis: Import data from Storage Container into analytics services like Snowflake for predictive maintenance, device efficiency evaluation, and other data analysis services.
+
+## Features and Advantages
+
+Using Azure Blob Storage data integration in EMQX can bring the following features and advantages to your business:
+
+- **Message Transformation**: Messages can undergo extensive processing and transformation in EMQX rules before being written to Azure Blob Storage, facilitating subsequent storage and use.
+- **Flexible Data Operations**: With the Azure Blob Storage Sink, specific fields of data can be conveniently written into Azure Blob Storage containers, supporting the dynamic setting of containers and object keys for flexible data storage.
+- **Integrated Business Processes**: The Azure Blob Storage Sink allows device data to be combined with the rich ecosystem applications of Azure Blob Storage, enabling more business scenarios like data analysis and archiving.
+- **Low-Cost Long-Term Storage**: Compared to databases, Azure Blob Storage offers a highly available, reliable, and cost-effective object storage service, suitable for long-term storage needs.
+
+These features enable you to build efficient, reliable, and scalable IoT applications and benefit from business decisions and optimizations.
+
+## Before You Start
+
+This section introduces the preparations required before creating an Azure Blob Storage Sink in EMQX.
+
+### Prerequisites
+
+- Understanding of [rules](./rules.md).
+- Understanding of [data integration](./data-bridges.md).
+
+### Create a container in Azure Storage
+
+1. To access Azure Storage, you'll need an Azure subscription. If you don't already have a subscription, create a [free account](https://azure.microsoft.com/free/) before you begin.
+
+2. All access to Azure Storage takes place through a storage account. For this quickstart, create a storage account using the [Azure portal](https://portal.azure.com/), Azure PowerShell, or Azure CLI. For help creating a storage account, see [Create a storage account](https://learn.microsoft.com/en-us/azure/storage/common/storage-account-create).
+
+3. To create a container in the Azure portal, navigate to your new storage account in the Azure portal. In the left menu for the storage account, scroll to the Data storage section, then select Containers. Select the + **Container** button, use `iot-data` as a name for your new container, and click **Create** to create the container.
+
```

Comment:
The name of the container shows "emqx" in the screenshot. We either need to change the screenshot or the container name. @id